bom/runtime/pom.xml

```diff
@@ -43,6 +43,13 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Debezium Outbox Extension -->
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-quarkus-outbox</artifactId>
+                <version>${debezium-outbox-quarkus.version}</version>
```

Comment:
Minor nitpick but the name of the property is not in line with the artifact name. Could you make them consistent?